### PR TITLE
Media manager: Add return values in order to fail loudly

### PR DESCRIPTION
**Pull request by pgivel**
## Description

This PR adds return values to most functions in media manager in order to communicate with users of this class if we failed to initialise video or audio input.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I've read & comply with the [contributing guidelines](https://github.com/Genymobile/genymotion-device-web-player/blob/main/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] I have made corresponding changes to the documentation (README.md).
- [x] I've checked my modifications for any breaking changes.

### CODE REVIEW

**Review by skornous — APPROVED** [review]
lgtm

**Review by JulienBolard — APPROVED** [review]

File: src/plugins/MediaManager.js [hunk 1]
```diff
@@ -130,9 +133,10 @@ module.exports = class MediaManager {
             log.debug('Client audio stream ready');
             this.audioStreaming = true;
             this.localAudioStream = mediaStream;
-            this.addAudioStream(mediaStream);
+            return this.addAudioStream(mediaStream);
```
Comment (jparez) [inline src/plugins/MediaManager.js:136]: if addAudioStream fail the error will be uncaught. `this.instance.renegotiateWebRTCConnection();` call renegotiateWebRTCConnection wihch do `this.peerConnection.createOffer` and createOffer return a promise.

I think `addAudioStream` need to return `this.instance.renegotiateWebRTCConnection();` or a Promise.reject() and this line should be prefix by an await.

File: src/plugins/MediaManager.js [hunk 2]
```diff
@@ -156,28 +160,32 @@ module.exports = class MediaManager {
...
+     * @returns {boolean} true on success, false on fail
      */
     stopVideoStreaming() {
         if (!this.videoStreaming) {
-            return;
+            return false;
         }
         log.debug('removed local video stream');
-        this.removeVideoStream(this.localVideoStream);
+        const result = this.removeVideoStream(this.localVideoStream);
```
Comment (jparez) [inline src/plugins/MediaManager.js:170]: Same thing here, This call renegotiateWebRTCConnection which call this.peerConnection.createOffer.

File: src/plugins/MediaManager.js (hunk 1 above)
Comment (pgivel) [reply src/plugins/MediaManager.js:136]: I was mostly focused on errors related to permissions or availability of input devices but you're right I should cover this case too. Thanks !

Comment (pgivel) [reply src/plugins/MediaManager.js]: Is it OK for you now ?

**Pull request merged**